src/app/shared/components/frame/frame.component.html

```diff
@@ -0,0 +1,3 @@
+<section class="py-20 mx-5 sm:mx-20 lg:mx-40">
+  <ng-content />
```

Comment:
Where is the h2? With the previous comment I meant to put a title property in a VM and make a permanent h2 in the html code where u can use this vm property as your purpose is to make a standardized section.